pkg/stdlib/strings/json.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // JSONParse returns a FQL value described by the JSON-encoded input string.
 // @params text (String) - The string to parse as JSON.
-// @returns FQL value (Read)
+// @returns (Read) - Returns FQL Value
```

Comment:
``(Value)``.
I think ``GoLand`` renamed it to ``Read`` during one of refactorings :)